frontend/src/lib/lemon-ui/LemonSelectMultiple/LemonSelectMultiple.stories.tsx

```diff
@@ -80,7 +78,28 @@ Loading.parameters = {
 
 export const NoOptions: Story = Template.bind({})
 NoOptions.args = {
-    mode: 'multiple-custom',
+    mode: 'multiple',
+    allowCustomValues: true,
     placeholder: 'No options...',
     options: [],
 }
+
+export const SingleOption: Story = Template.bind({})
+SingleOption.args = {
+    mode: 'single',
+    placeholder: 'Only one option allowed',
+}
```

Comment:
Not sure if I was misunderstanding the API but I shouldn't be able to add options here right?
https://github.com/PostHog/posthog/assets/6685876/3d71219c-bd20-43e1-8f2f-d6b2c50be387

Reply:
Should we also be removing the custom value from the list when it is deleted from the input?

Reply:
Yeah this one was a bug - the .args was applied to the wrong story